assets/sass/components/_summary-card.scss

```diff
@@ -0,0 +1,20 @@
+.hmpps-summary-card {
+  border: 1px solid black;
+  // margin-bottom: 10px;
+
+  .hmpps-summary-card__header {
+    // padding: 0 15px;
+    background-color: #F3F2F1;
```

Comment:
Please add this govuk colour too. Or it can be done in the CSS clean up task.
govuk-colour("light-grey")